ipyvizzu.py

```diff
@@ -126,15 +126,36 @@ def dump(self):
 
 class Chart:
     """
-    Wrapper over vizzu Chart
+    Wrapper over Vizzu Chart
     """
 
-    def __init__(
-        self, vizzu_url="https://cdn.jsdelivr.net/npm/vizzu@latest/dist/vizzu.min.js"
-    ):
-        self._vizzu_url = vizzu_url
+    def __init__(self):
+        self._vizzu = "https://cdn.jsdelivr.net/npm/vizzu@latest/dist/vizzu.min.js"
+        self._div_style = "width:800px; height:480px;"
         self._calls = []
 
+    @property
+    def vizzu(self):
+        """
+        Url of Vizzu
+        """
+        return self._vizzu
+
+    @vizzu.setter
+    def vizzu(self, url):
```

Comment:
Wouldn't be better to have some preset value for vizzu source instead of a free style str?

```python
import enum

class VizzuSource(str, enum.Enum):
    latest = "https://cdn.jsdelivr.net/npm/vizzu@latest/dist/vizzu.min.js"
    ...

class Chart:
    def set_vizzu(self, source: VizzuSource):
        self._vizzu_url = source
```

Reply:
Enums are great for our urls, but I think free style is useful if someone wants to load Vizzu from its own environment.
(I can’t estimate how much demand there will be for this.)

Reply:
Shouldn't vizzu version be fixed for an ipyvizzu version? There could be incompatible changes on Vizzu API between versions.

Reply:
I think ipyvizzu should support only a restricted set of versions. The local vizzu would be part of the ipyvizzu project and `Chart` should evaluate the `vizzu=VizzuSource.local` in a proper way.

Reply:
I close this conversation, because vizzu url could be changed already in a previous PR - but I open a new issue for this to resolve this problem in an another PR.